apps/src/code-studio/components/aiTutor/validationTutor.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Button from '@cdo/apps/templates/Button';
+import style from './ai-tutor.module.scss';
+import {askAITutor} from '@cdo/apps/aiTutor/redux/aiTutorRedux';
+import {useSelector, useDispatch} from 'react-redux';
+
+// AI Tutor feature that explains to students why their code is not passing tests.
+const ValidationTutor = () => {
+  const dispatch = useDispatch();
+  const javalabState = useSelector(state => state.javalab);
+  const javalabEditorState = useSelector(state => state.javalabEditor);
+  const studentCode =
+    javalabEditorState.sources[
+      javalabEditorState.fileMetadata[javalabEditorState.activeTabKey]
+    ].text;
+  const aiTutorState = useSelector(state => state.aiTutor);
+  const systemPrompt =
+    'You are a tutor in a high school computer science class. Students in the class are studying Java and they would like to know in age-appropriate, clear language why their tests are not passing.';
+
+  const handleSend = async studentCode => {
+    dispatch(
+      askAITutor({systemPrompt: systemPrompt, studentCode: studentCode})
+    );
```

Comment:
- [ ] 🤔 openAi also needs access to the test file(s)... find out where those live and how to send them along in the request

Reply:
Are you planning to add that within the scope of this PR?

Reply:
No - this was a note to myself. I'll disable the button for now and build out sending the test files along with the request in the next PR. Thanks for checking!

Reply:
That work is happening here: https://github.com/code-dot-org/code-dot-org/pull/55076